### PR TITLE
Elevator door fix and transition visuals

### PR DESCRIPTION
**Pull request by marredal**
Added transition visuals between elevator and voltage puzzle.
Fixed doors going into infinity.

### CODE REVIEW

File: Game/Resources/elevatorScene/Scripts/elevator_backDoorCon.as [hunk 1]
```diff
@@ -13,6 +13,7 @@ class elevator_backDoorCon {
         @doorWithX = GetEntityByGUID(1511870598);
         @doorWithService = GetEntityByGUID(1511870668);
         openDoor = false;
+		openingDoor = false;
```
Comment (NiceGuyAndersson) [inline Game/Resources/elevatorScene/Scripts/elevator_backDoorCon.as]: Tabs. Also, isn't the declaration of this variable removed?

**Comment by marredal** [comment]
wait with pushing this

**Comment by marredal** [comment]
Okay it is ready to be reviewed again

File: Game/Resources/elevatorScene/Scripts/elevator_backDoorCon.as [hunk 2]
```diff
@@ -61,6 +60,7 @@ class elevator_backDoorCon {
     }
 
     void CloseDoor() {
+        openingDoor = true;
```
Comment (andens) [inline Game/Resources/elevatorScene/Scripts/elevator_backDoorCon.as]: Is this an oversight? It seems wrong to set `openingDoor` to true when calling `CloseDoor`.

Comment (marredal) [reply Game/Resources/elevatorScene/Scripts/elevator_backDoorCon.as]: umm

**Comment by marredal** [comment]
iamheman

**Pull request merged**